Searching/LinearSearch.cpp

```diff
@@ -0,0 +1,49 @@
+/* 
+    Linear search:
+    A search algorithm that walks sequentially through a list to find
+    the position of a target value within a vector. 
+
+    Time complexity: O(n), where n is the number of elements in the vector
+    Space complexity: O(1), i.e. the algorithm does not require additional space
+*/
+
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+int linearSearch(const int value, const vector<int>& searchVect) {
+    
+    for (unsigned int i = 0; i < searchVect.size(); i++) {
+        if (searchVect.at(i) == value) {
+            return i; // return the element in searchVect where value is found
```

Comment:
Comment should just be:
```C++
// index at which the value is found
```
as it would add meaning to the code while being short.